airflow/models/taskinstance.py

```diff
@@ -1124,24 +1085,19 @@ def get_triggering_events() -> dict[str, list[AssetEvent | AssetEventPydantic]]:
         "outlet_events": OutletEventAccessors(),
         "ds": ds,
         "ds_nodash": ds_nodash,
-        "execution_date": logical_date,
+        "logical_date": logical_date,
         "expanded_ti_count": expanded_ti_count,
         "inlets": task.inlets,
         "inlet_events": InletEventsAccessors(task.inlets, session=session),
-        "logical_date": logical_date,
         "macros": macros,
         "map_index_template": task.map_index_template,
-        "next_ds": get_next_ds(),
-        "next_ds_nodash": get_next_ds_nodash(),
-        "next_execution_date": get_next_execution_date(),
+        "next_logical_date": get_next_logical_date(),
```

Comment:
This one is also deprecated and should be removed.

Reply:
I have removed: prev_logical_date_success, prev_logical_date and next_logical_date.